docs/ios/open-source/manual-instrumentation/custom-spans.md

```diff
@@ -0,0 +1,253 @@
+---
+title: Custom Spans
+description: Track the duration and performance of custom operations in your iOS app
+sidebar_position: 1
+---
+
+# Custom Spans
+
+Custom spans allow you to measure the duration of specific operations in your app, providing insights into performance and behavior of code paths that matter to your business.
+
+## What are Spans?
+
+In OpenTelemetry, spans represent a unit of work or operation. Each span:
+- Has a name and type
+- Tracks when the operation started and ended
```

Comment:
would note 
- they MUST end (even if our SDK accounts for certain scenarios where instrumented spans aren't ended) 
- and the endSpan records whether they were successful, failure, etc